migrations/v7.js

```diff
@@ -0,0 +1,135 @@
+  const originalInstruction = '';
+  whereFromPlugin('Matching - from v6.0.0', { name: 'adapt-contrib-matching', version: '<7.2.0' });
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - update instruction attribute', async content => {
+    matchings.forEach(matching => {
+      if (matching.instruction === originalInstruction) {
+        matching.instruction = 'Choose an option from each dropdown list and select Submit.';
+      }
+    });
+    return true;
+  });
+  checkContent('Matching - check instruction attribute', async content => {
+    const isValid = matchings.every(({ instruction }) => instruction !== originalInstruction);
+    if (!isValid) throw new Error('Matching - instruction attribute invalid');
+    return true;
+  });
+  updatePlugin('Matching - update to v7.2.0', { name: 'adapt-contrib-matching', version: '7.2.0', framework: '>=5.19.1' });
+});
+
+describe('Matching - v7.2.0 to v7.2.1', async () => {
+  whereFromPlugin('Matching - from v7.2.0', { name: 'adapt-contrib-matching', version: '<7.2.1' });
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - add ariaQuestion attribute', async content => {
+    matchings.forEach(matching => (matching.ariaQuestion = ''));
+    return true;
+  });
+  checkContent('Matching - check ariaQuestion attribute', async content => {
+    const isValid = matchings.every(matching => matching.ariaQuestion === '');
+    if (!isValid) throw new Error('Matching - ariaQuestion attribute invalid');
+    return true;
+  });
+  updatePlugin('Matching - update to v7.2.1', { name: 'adapt-contrib-matching', version: '7.2.1', framework: '>=5.19.1' });
+});
+
+describe('Matching - v7.2.1 to v7.3.0', async () => {
+  whereFromPlugin('Matching - from v7.2.1', { name: 'adapt-contrib-matching', version: '<7.3.0' });
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - add _isRandomQuestionOrder attribute', async content => {
+    matchings.forEach(matching => (matching._isRandomQuestionOrder = false));
+    return true;
+  });
+  checkContent('Matching - check _isRandomQuestionOrder attribute', async content => {
+    const isValid = matchings.every(matching => matching._isRandomQuestionOrder === false);
+    if (!isValid) throw new Error('Matching - _isRandomQuestionOrder attribute invalid');
+    return true;
+  });
+  updatePlugin('Matching - update to v7.3.0', { name: 'adapt-contrib-matching', version: '7.3.0', framework: '>=5.19.1' });
+});
+
+describe('Matching - v7.3.0 to v7.4.1', async () => {
```

Comment:
The changes in this block occur from `v7.4.0` to `v7.4.1`

FYI these two new attributes have not been added to `properties.schema`

Reply:
> FYI these two new attributes have not been added to properties.schema

Thanks, @chris-steele . I actually do see [`_hasItemScoring`](https://github.com/adaptlearning/adapt-contrib-matching/blob/d2bee9f5c25eb8b3074dc66d818792389e344f5d/properties.schema#L172) and [`_allowOnlyUniqueAnswers`](https://github.com/adaptlearning/adapt-contrib-matching/blob/d2bee9f5c25eb8b3074dc66d818792389e344f5d/properties.schema#L181) in properties.schema?

Reply:
Apologies, yes I see `_hasItemScoring` and `_allowOnlyUniqueAnswers` arrived in v6.0.0, but weren't added to the modern schemas until `v7.4.1` 😄